File: tests/nodejs/createThread.js
var request = require('request');

var jsonBody = {
	subdomain_id: 1,
	title: "some title",
	author: "admin",
	context: "some context"
}

request({
	url: 'http://localhost:3000/createThread',
	method: 'post',
	json: true,
	body: jsonBody
}, function (error, response, body) {
	console.log(response);
});

File: tests/python/thread/createThread.py
import requests
import json
import sys

jsonBody = {
	"subdomain_id": 1,
	"title": "new title",
	"author": "admin",
	"context": "new context"
}

headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
res = requests.post('http://localhost:3000/createThread', data = json.dumps(jsonBody), headers = headers)
print(res.json())